ui/tests/e2e/search_log.test.ts

```diff
@@ -9,6 +9,9 @@ describe('Search Logs', () => {
 
       await Promise.all([page.waitForNavigation(), page.click('a#search_logs')])
 
+      // FIXME: should be integrated with ngm
```

Comment:
How about introducing a special mode that does not check ngm at all or something like that? It may be common for E2E tests to disable user layers (like first-time teaching bubbles).